setup.py

```diff
@@ -243,11 +245,11 @@ def write_version(filename: str = os.path.join(*[my_dir, "airflow", "git_version
 flask_oauth = [
     'Flask-OAuthlib>=0.9.1,<0.9.6',  # Flask OAuthLib 0.9.6 requires Flask-Login 0.5.0 - breaks FAB
     'oauthlib!=2.0.3,!=2.0.4,!=2.0.5,<3.0.0,>=1.1.2',
-    'requests-oauthlib==1.1.0',
+    'requests-oauthlib<1.2.0',
 ]
 google = [
     'PyOpenSSL',
-    'google-ads>=4.0.0',
+    'google-ads>=4.0.0,<8.0.0',
```

Comment:
```suggestion
    'google-ads>=3.3.0,<8.0.0',
```
Should support 3.6 and it has v2 API

Reply:
Does 4.0 drop support for Py 3.6?

If so we'll need a more complex thing here including `; python_version...`

Reply:
> If so we'll need a more complex thing here including

Is it necessary? Won't pip resolver use the right version for each python?

Reply:
Ahahahahahaah.

No, don't think so :(

Reply:
The 8.0.0 limitation is needed, because the Ads operators are using v2 version of the API and support for those have been removed in 8.0.0 (very recently - in November). The operator should be updated (but not now).

The 4.0.0 still supports 3.6.7+. Support for 3.7 has been removed in one of the 4.1.* versions. PYPI solves it nicely on their own as those non-3.7 packages have the limitation to Python>=3.7 set and pypi will not go beyond that version in 3.6.

Reply:
See here: 4.0.0 still supports 3.6.7+  https://pypi.org/project/google-ads/4.0.0/

setup.py:
```
    python_requires='>=3.6',
```

Reply:
Oh I'm confusing the case where we don't want the module for a python version at all. Sorry!